Integrations/integration-SumoLogic.yml

```diff
@@ -259,29 +296,32 @@ script:
       description: 'The ISO 8601 date of the time range to end the search. For example:
         2016-08-28T12:00:00). Can also be milliseconds since epoch.'
     - name: limit
-      description: Maximum number of results to return from the query. Default is 100. The value specified will override the default set in the "limit" parameter.
+      description: Maximum number of results to return from the query. Default is
+        100. The value specified will override the default set in the "limit" parameter.
       defaultValue: "100"
     - name: offset
       description: Return results starting at this offset. should be int - by default
         is 0
       defaultValue: "0"
     - name: timezone
-      description: The time zone if from/to is not in milliseconds. Default is UTC. See
-        this (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) article
+      description: The time zone if from/to is not in milliseconds. Default is UTC.
+        See this (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) article
         for a list of time zone codes.
       defaultValue: UTC
     - name: maxTimeToWaitForResults
-      description: Maximum amount of time (in minutes) to wait for the search to complete. Default is 10
-        minutes.
+      description: Maximum amount of time (in minutes) to wait for the search to complete.
+        Default is 10 minutes.
       defaultValue: "10"
     - name: headers
-      description: 'A CSV list of table headers that are displayed in order. For example: "_blockid,_collector,_format".'
+      description: 'A CSV list of table headers that are displayed in order. For example:
```

Comment:
A comma-separated list of table headers that are displayed in order. For example: